src/components/Editable.tsx

```diff
@@ -535,7 +541,7 @@ const Editable = ({ disabled, isEditing, simplePath, path, cursorOffset, showCon
         path,
         text: isHTML(plainText)
           ? plainText
-          : htmlText || plainText,
+          : isCopiedFromApp(htmlText) ? plainText : htmlText || plainText,
```

Comment:
We need to preserve formatting such as `<b>` and `<i>`, so we can't just take the `plainText`.